playbooks/roles/tinymce_plugins/tasks/import_tinymce_plugin.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: Add additional tinymce plugins
+  block:
+    - name: Clone plugin
+      git:
+        repo: "{{ plugin.repo }}"
+        dest: "{{ tinymce_plugin_temp_dir }}/{{ plugin.name }}"
+        version: "{{ plugin.version | default('master') }}"
+        accept_hostkey: true
+        key_file: "{% if EDXAPP_USE_GIT_IDENTITY %}{{ edxapp_git_identity }}{% endif %}"
+    - name: Move plugin to tinymce plugins directory
+      command: "cp -r {{ tinymce_plugin_temp_dir }}/{{ plugin.name }}{{ plugin.plugin_path | default('') }} {{ tinymce_plugins_dir }}"
```

Comment:
@nizarmah  Not really a problem, but why not clone directly into the `tinymce_plugins_dir`, instead of cloning to a temp one and then copying over?

Reply:
@pkulkark because sometimes the files/directories we want to pull into the `tinymce_plugins_dir` differs.

So for example, the `adsklink` plugin shown here contains a `README.md` which we don't want to move to the `tinymce_plugins_dir`.

Another example is [tinymce-codemirror](https://gitlab.com/tinymce-plugins/tinymce-codemirror) and the plugin in this repo is inside `plugins/codemirror`.
By specifying what we want to move from the cloned plugin, there won't be a mess.

Hopefully that answers your question :+1:

Reply:
@nizarmah Ah I missed the `plugin_path` in the command. Makes sense :+1: